tests/components/component.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import TinyComponent from './tiny-component';
+
+export default class TestComponent extends React.Component {
+  constructor(...args) {
+    super(...args)
+    this.state = {}
+  }
+
+  render() {
+    return <body>
+    <div onClick={this.props.onClick}>{this.state.test}</div>
```

Comment:
This changes mysteriously breaks some tests unrelated to this pull request.
Maybe @zackify can take a look